DTS-STN/Service-Canada-Labs
# reverting dependencies to v1.5.1

# Description

One of the snyk-recommended updates to our dependencies broke french translations when running `yarn dev`. While they were all very minor changes, nonetheless they broke the application when running `yarn dev`.

Will be looking into a way to automatically test for these kinds of issues in the future.

## Acceptance Criteria

When running `yarn dev`, the application should not crash when toggling to French from English.

## Test Instructions

1. On main, run `yarn install` then `yarn dev` and attempt toggling to French from English on any page. You should see an error.
2. Switch to `revert-dependencies` branch and delete node_modules.
3. Run `yarn install` then `yarn dev`
4. Try toggling to French from English and you should not see the error.
